src/main/TestRunner.java

```diff
@@ -0,0 +1,24 @@
+package main;
+
+
+import main.exceptions.CsvLineException;
+
+import java.util.Comparator;
+
+public class TestRunner {
+    public static void main(String[] args) throws CsvLineException {
```

Comment:
it's not a unit testing. 
unit testing is strongly required for software development.

i advise you to take the course [Java: Testing with JUnit](https://learn.epam.com/detailsPage?id=a2f13cb6-109d-4632-951d-d074a4ab95af) on the learn platform.